classification2.Rmd

```diff
@@ -1012,7 +1050,7 @@ one over increasing predictor set sizes
 and another to check which predictor to add in each round (where you see `for (j in 1:length(names))` below).
 For each set of predictors to try, we construct a model formula,
 pass it into a `recipe`, build a `workflow` that tunes
-a KNN classifier using 5-fold cross-validation, 
+a $K$-NN classifier using 5-fold cross-validation, 
```

Comment:
I thought I saw that you were changing to K-nearest neighbors everywhere. I might have missed something though, so take with a grain of salt.